packages/cosmic-swingset/check-validator.js

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env node
```

Comment:
This looks OK at a skim; I should test it, but I'll take the risk that it's OK instead.